tests/kafkatest/services/kafka/kafka.py

```diff
@@ -34,17 +34,22 @@
 class KafkaService(JmxMixin, Service):
 
     PERSISTENT_ROOT = "/mnt"
-    LOG4J_CONFIG_FILE = os.path.join(PERSISTENT_ROOT, "kafka-log4j.properties")
+    STDOUT_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
+    STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
```

Comment:
Is this duplication intentional or did you want stderr on a different output? Usually for Kafka you wouldn't need to manage them separately because everything *should* be going through log4j and have a relevant log level setting.